lms/resources/_js_config.py

```diff
@@ -42,6 +43,24 @@ def _auth_token(self):
             self._request.lti_user
         )
 
+    @property
+    def _debug(self):
+        """
+        Return some debug information.
+
+        Currently used in the Gherkin tests.
+        """
+        debug_info = {}
+
+        if self._request.lti_user:
```

Comment:
Apparently there are some LTI launch requests where `request.lti_user` is `None`. Checks for `request.lti_user` being `None` already exist elsewhere in the `JSConfig` code, and the tests cover this case and will crash if it's not handled.

I'm not sure that this is actually necessary / actually happens. But not 100% sure that it doesn't either. So sticking with it for now